src/main/java/seedu/address/ui/HelpWindow.java

```diff
@@ -248,6 +271,15 @@ public HelpWindow(Stage root) {
             commandText.setText(commandString);
 
             // Setting the corresponding MenuItem to scroll to the Text object
+            ChangeListener<Number> listener = (obs, oldText, newText) -> {
+                Platform.runLater(() -> {
+                    commandText.setWrappingWidth(getRoot().getScene().getWidth() - 112);
```

Comment:
Perhaps avoid having magic number such as "112" as assign it to an int variable to show what it is about?